arangod/Transaction/Manager.cpp

```diff
@@ -664,6 +724,11 @@ Result Manager::updateTransaction(TransactionId tid, transaction::Status status,
     mtrx.type = MetaType::Tombstone;
     mtrx.updateExpiry();
     mtrx.finalStatus = status;
+    if (state->numCommits() > 0) {
+      // note that we have performed a commit or an intermediate commit.
+      // this is necessary for follower transactions
+      mtrx.intermediateCommits = true;
+    }
```

Comment:
isn't the idea that you cann abort follower transactions with intermediate commits